src/Defaults.php

```diff
@@ -84,7 +84,19 @@ private static function getScrubFields()
     
     public function sendMessageTrace($sendMessageTrace = null)
     {
-        return $sendMessageTrace ? $sendMessageTrace : $this->defaultSendMessageTrace;
+        return $sendMessageTrace !== null ? $sendMessageTrace : $this->defaultSendMessageTrace;
+    }
+    
+    public function captureErrorStacktraces($captureErrorStracktraces = null)
+    {
+        return $captureErrorStracktraces !== null ?
+            $captureErrorStracktraces :
+            $this->defaultCaptureErrorStacktraces;
+    }
+    
+    public function localVarsDump($localVarsDump = null)
+    {
+        return $localVarsDump ? $localVarsDump : $this->defaultLocalVarsDump;
```

Comment:
I changed this. `$localVarsDump ? $localVarsDump : $this->defaultLocalVarsDump` is buggy